pkg/epp/scheduling/config.go

```diff
@@ -42,15 +40,3 @@ type SchedulerConfig struct {
 	picker              plugins.Picker
 	postSchedulePlugins []plugins.PostSchedule
 }
-
-// When the scheduler is initialized with NewScheduler function, this config will be used as default.
-// it's possible to call NewSchedulerWithConfig to pass a different argument.
-
-// For build time plugins changes, it's recommended to change the defaultConfig variable in this file.
-var defaultConfig = &SchedulerConfig{
-	preSchedulePlugins:  []plugins.PreSchedule{},
-	filters:             []plugins.Filter{&filter.SheddableRequestFilter{}, filter.LowLatencyFilter},
-	scorers:             map[plugins.Scorer]int{},
-	picker:              &picker.RandomPicker{},
-	postSchedulePlugins: []plugins.PostSchedule{},
-}
```

Comment:
after having some experience with setting a custom plugins, the recommended way would be to set in main.go the scheduler using NewSchedulerWithConfig. 
no need for global var. default configuration is defined inside NewScheduler instead.